sycl/include/CL/sycl/handler.hpp

```diff
@@ -996,11 +996,8 @@ class handler {
       range<Dims> Range = Src.get_range();
       parallel_for< class __copyAcc2Ptr< T_Src, T_Dst, Dims, AccessMode,
                                          AccessTarget, IsPlaceholder>>
-                                         (Range, [=](id<Dims> Index) {
-        size_t LinearIndex = Index[0];
-        for (int I = 1; I < Dims; ++I)
-          LinearIndex += Range[I] * Index[I];
-        ((T_Src *)Dst)[LinearIndex] = Src[Index];
+                                         (Range, [=](item<Dims, false> Index) {
```

Comment:
The commit message now seems confusing, could you please update it?

Reply:
Updated.